File: 1020.py
tempo = int(input())
anos = (tempo//365)
meses = (tempo%365)//30
dias = ((tempo%365)%30)

print('{} ano(s)'.format(anos))
print('{} mes(es)'.format(meses))
print('{} dia(s)'.format(dias))

File: 1020.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
    
var tempo = parseInt(lines.shift());

var anos = Math.floor(tempo/365);
var meses = Math.floor((tempo%365)/30);
var dias = Math.floor((tempo%365)%30);
console.log(anos + ' ano(s)');
console.log(meses + ' mes(es)');
console.log(dias + ' dia(s)');